### PR TITLE
475 adding resize by width and quality filter

### PR DESCRIPTION
**Pull request by rgritti**

### CODE REVIEW

File: eskip/sample.eskip [hunk 1]
```diff
@@ -31,4 +34,10 @@ cropByHeight: Path("/images/cropbyheight/:image")
   -> cropByHeight(1000, "south")
   -> "http://localhost:9090";
 
+widthAndquality: Path("/images/waq/:image")
```
Comment (danpersa) [inline eskip/sample.eskip]: `widthAndQuality` I would capitalize quality

File: filters/resizebywidth.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+func (r *resizeByWidth) CreateOptions(image *bimg.Image) (*bimg.Options, error) {
+	log.Debug("Create options for resize by width ", r)
+
+	imgSize, err := image.Size()
+
+	if err != nil {
+		return nil, err
+	}
+
+	// resize only if the original width is bigger than the required width
```
Comment (danpersa) [inline filters/resizebywidth.go]: why not allowing enlarging the image?

File: filters/resizebywidth_test.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+	assert.Equal(t, 256, options.Width)
+}
+
+func TestResizeByWidth_CreateFilter(t *testing.T) {
+	imagefiltertest.TestCreate(t, NewResizeByWidth, []imagefiltertest.CreateTestItem{{
+		Msg:  "no args",
+		Args: nil,
+		Err:  true,
+	}, {
```
Comment (danpersa) [inline filters/resizebywidth_test.go]: you should have tests to test the image size conditions. If the image's withs is bigger than the requested width, etc

Comment (rgritti) [inline filters/resizebywidth_test.go]: I removed that now

**Pull request closed without merging**